.circleci/config.yml

```diff
@@ -43,10 +46,26 @@ jobs:
     steps:
       - checkout
       - read_cache
+      - node/install
+
+      - run:
+          name: Install protobufjs
+          command: |
+            npm install protobufjs@6.11.3
+
+      - run:
+          name: Install long.js
+          command: |
+            npm install long@5.2.0
+
+      - run:
+          name: Install Firefox
+          command: |
+            apk add firefox
 
       - run:
           name: Build and test
-          command: ./gradlew clean check publishToIntegrationRepository --stacktrace
+          command: ./gradlew clean check publishToIntegrationRepository --stacktrace --refresh-dependencies
```

Comment:
what does refresh-dependencies do?

Reply:
I must have been having some problem getting stale dependencies. I can probably get rid of it now.